### PR TITLE
Snippets

### PR DESCRIPTION
**Pull request by RyanVillenaUCR**
Make snippets using k-means clustering.

Please pull and actually test with your own scrape, I'm not sure that the output this gives is correct.

Closes #11.

### CODE REVIEW

File: src/main/java/voyager/SnippetFactory.java [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+        stopwords.add("too");
+        stopwords.add("very");
+        stopwords.add("s");
+        stopwords.add("t");
+        stopwords.add("can");
+        stopwords.add("will");
+        stopwords.add("just");
+        stopwords.add("don");
+        stopwords.add("should");
+        stopwords.add("now");
```
Comment (MrFlynn) [inline src/main/java/voyager/SnippetFactory.java:553]: Couldn't we just initialize this using an array + the hashset constructor? Seems a lot less complicated and probably has a lot less startup cost compared to this method.

Comment (RyanVillenaUCR) [reply src/main/java/voyager/SnippetFactory.java:553]: It'd be syntactically better, but would it really be cheaper? The HashSet ctor requires a Collection not an array, so we'd need to dump the array into a list using Arrays.asList(), and I don't know if that'd be cheaper than just adding to the HashSet like we do now.

Comment (RyanVillenaUCR) [reply src/main/java/voyager/SnippetFactory.java]: Changed my mind, even if performance is similar the readability is enough of a plus for this to happen. Commit c82da18 fixes this.

**Comment by MrFlynn** [comment]
Closes #11.

**Pull request merged**